cmd/authelia-scripts/migration_local.go

```diff
@@ -102,7 +102,7 @@ func migrateLocalU2FSecret(dbProvider storage.Provider) {
 }
 
 func migrateLocalPreferences(dbProvider storage.Provider) {
-	file, err := os.Open(path.Join(localDatabasePath, "prefered_2fa_method"))
+	file, err := os.Open(path.Join(localDatabasePath, "preferred_2fa_method"))
```

Comment:
This change will break the migration script.

Reply:
Ah so the original store was misspelled?
I'll revert this and the mongo one too.